### PR TITLE
Composer cleanup and update

### PR DESCRIPTION
**Pull request by tmatsuo**

### CODE REVIEW

**Comment by bshaffer** [comment]
Unless there's a reason for using tilde instead, we should use `google/apiclient:^2.0`, as this is more strict to semantic versioning.

File: appengine/flexible/analytics/composer.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
         "symfony/css-selector": "^3.1"
     },
     "require-dev": {
-        "phpunit/phpunit": "~4",
-        "google/cloud-tools": "^0.1.0"
+        "google/cloud-tools": "<=1.0",
```
Comment (bshaffer) [inline appengine/flexible/analytics/composer.json]: This goes against semantic versioning, are you sure we don't want to just tag this to a specific version?

Comment (tmatsuo) [inline appengine/flexible/analytics/composer.json]: I actually want to auto update cloud-tools to the latest until it hits 1.0

**Comment by tmatsuo** [comment]
The e2e tests are also passing on our jenkins!

**Comment by bshaffer** [comment]
LGTM!

**Pull request merged**